rbc/irtools.py

```diff
@@ -8,6 +8,11 @@
 from .targetinfo import TargetInfo
 from .npy_mathimpl import *  # noqa: F403, F401
 
+
+def lf(l):
+    return l + [e + 'f' for e in l] + [e + 'l' for e in l]
```

Comment:
Make `lf` private: rename it to `_lf`.
Suggest not using `l` as variable name (as it looks like `1`). Use `lst`, for instance.